CHANGELOG.md

```diff
@@ -310,4 +311,5 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
 [PR #973]: https://github.com/bareos/bareos/pull/973
 [PR #982]: https://github.com/bareos/bareos/pull/982
 [PR #996]: https://github.com/bareos/bareos/pull/996
+[PR #999]: https://github.com/bareos/bareos/pull/999
```

Comment:
These links are automatically updated during release and should not be updated, as the cause evern more conflicts.

Reply:
Done